json/products/mozilla-foundation-recurring-donation.json

```diff
@@ -0,0 +1,24 @@
+{
+  "description": {
+    "en": "Recurring Donation"
+  }, 
+  "img": "https://raw.githubusercontent.com/mozilla/payments-config/master/payments_config/assets/default.png", 
+  "auth_required": false, 
+  "price": {}, 
+  "email_required": true, 
+  "seller": {
+    "kind": "donations", 
+    "terms": "http://pay.dev.mozaws.net/terms/", 
+    "name": {
+      "en": "Mozilla Foundation"
+    }, 
+    "url": "http://pay.dev.mozaws.net/", 
+    "email": "support@foundation.mozilla.org", 
+    "id": "mozilla-foundation"
+  }, 
+  "recurrence": "monthly", 
+  "currency": "USD", 
+  "amount": null, 
```

Comment:
huh. I'm really confused by this. I see that you probably copied it from the one-time donation config but I don't know if this is in use because we already have `price`.